app/assets/stylesheets/alchemy/tables.scss

```diff
@@ -93,12 +93,6 @@ tr.even td {
   white-space: nowrap;
   @extend .disable-user-select;
 
-  a {
```

Comment:
This selector never hits, because the `.icon` class on all of the core modules already adds margin. I double checked it and I think this can be removed.